producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -180,26 +210,53 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.presentationTs = frame.decodingTs;
             frameIndex++;
         }
+        CHK_LOG_ERR(retStatus);
+
+        if (frame.frameData != NULL) {
```

Comment:
No need to check for framedata being null with safe_memfree

Reply:
My change did not go in. But, now it should be visible.